src/app/cli/src/commands/logout_command.rs

```diff
@@ -33,15 +33,19 @@ impl LogoutCommand {
             server,
         }
     }
+
+    fn get_server_url(&self) -> Url {
+        self.server
+            .as_ref()
+            .cloned()
+            .unwrap_or_else(|| Url::parse(odf_server::DEFAULT_ODF_FRONTEND_URL).unwrap())
+    }
 }
 
 #[async_trait::async_trait(?Send)]
 impl Command for LogoutCommand {
     async fn run(&mut self) -> Result<(), CLIError> {
-        let odf_server_frontend_url = self
-            .server
-            .clone()
-            .unwrap_or_else(|| Url::parse(odf_server::DEFAULT_ODF_FRONTEND_URL).unwrap());
+        let odf_server_frontend_url = self.get_server_url();
```

Comment:
I was a bit confused when I ran:
```
kamu logout
```
and command produced no output while all tokens remained present. Took me a second to realize that it probably tries to log me out of some default server, which I'm not logged into.

Perhaps we can add a confirmation message, e.g.:
```
Logged out of <server>
```

Also a `logout --all` flag would be really nice to have.

Reply:
Done both